lua/orgmode/colors/from_config/priority.lua

```diff
@@ -0,0 +1,12 @@
+local config = require('orgmode.config')
+
+local M = {}
+
+function M.get_queries()
+  return vim.tbl_map(function(priority)
+    return string.format('((expr) @OrgPriority%s (#eq? @OrgPriority%s "[#%s]"))', priority, priority, priority)
```

Comment:
_todo_: Not sure if there is a better way to format in the same value multiple times w/ Lua's `string.format`.